src/styles/button.scss

```
@@ -0,0 +1,21 @@
+.button {
+  background-color: $button-blue-color;
+  color: white;
+  border: none;
+  border-radius: 5px;
+  height: 40px;
+  width: 166px;
+  text-decoration: none;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-align: center;
+  cursor: pointer;
+  text-transform: uppercase;
+}
+
+.button:hover {
+  background-color: white;
+  color: $button-blue-color;
+  border: 1px solid $button-blue-color;
+}
```

```suggestion
  text-transform: uppercase;
&:hover {
  background-color: white;
  color: $button-blue-color;
  border: 1px solid $button-blue-color;
}
}
```
make use of nesting
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**